common/include/opae/types.h

```diff
@@ -165,6 +165,8 @@ typedef void *fpga_event_handle;
  * an accelerator resource. The error API provides functions to retrieve these
  * information structures from a particular resource.
  */
+
+typedef void *fpga_object;
```

Comment:
This is inserted between fpga_error_info and its Doxygen comment.  Doxygen won't do the right thing in this case.  Move it above the comment or below the fpga_error_info definition.

Reply:
👍